app/src/main/java/com/shagalalab/constitution/article/ArticleFragment.kt

```diff
@@ -26,9 +27,15 @@ class ArticleFragment(private var chapterId: Int) :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         textView.movementMethod = ScrollingMovementMethod()
-        viewModel.getArticles(chapterId)
+        val s = SpannableStringBuilder()
+        if (check)
```

Comment:
You are not following the [kotlin style guide](https://developer.android.com/kotlin/style-guide#expressions), if you don't put braces, then fit in one line, otherwise use braces.